README.md

```diff
@@ -155,27 +155,52 @@ And lastly if you don't want to stop [`attribute inheritance`](https://v3.vuejs.
 
 ## Styling Wowerlay
 
-Styling wowerlay is too simple because `<Wowerlay/>` is just a single wrapper element. You can give any class any style and any attribute to it, that's why I didn't make props like `width`, `height` because you have full control with styles.
-
-`!!! Except necessary ones, please do not break them :)`
+Styling wowerlay is too simple because `<Wowerlay/>` is just a single wrapper element. You can give any class any style and any attribute to it, that's why it doesn't have props like `width`, `height` because you have full control with styles.
 
-By default `<Wowerlay/>` will try to fit in viewport. If you don't want that behavior in future you can use `disableFit: boolean` prop but not implemented yet.
+`!!! Except necessary styles, please do not break them :)`
 
 ```html
-<Wowerlay tag="something" style="width: 300px; height: 300px;"> Content Goes Here </Wowerlay>
+<Wowerlay tag="span" style="width: 300px; height: 300px; display: inline-block">
+  Content Goes Here
+</Wowerlay>
 ```
 
 ## Props
 
 ```ts
-export interface WowerlayBaseProps {
-  position?: 'left' | 'right' | 'top' | 'bottom'; // forced position for Wowerlay. default: bottom
-  verticalGap: number; // vertical gap between Wowerlay and target. default: 0
-  horizontalGap: number; // horizontal gap between Wowerlay and target. default: 0
-  fixed?: boolean; // if set wowerlay will not update it's position after mount
-  target?: HTMLElement; // target element for Wowerlay to follow
-  tag?: string; // tagName for Wowerlay container. default: 'div'
-  centered?: boolean; // if given wowerlay will be center itself in current position.
+interface WowerlayProps {
+  /**
+   * Forced position for Wowerlay.
+   * @default "bottom"
+   */
+  position: 'left' | 'right' | 'top' | 'top-right' | 'bottom' | 'bottom-right';
```

Comment:
Why `top-left` and `bottom-left` are not an option.

Reply:
Because default is left.

Reply:
but `left` is not `top-left` or `bottom-left`.

Reply:
By default `top`  and `bottom` places itself to `(top/bottom)-left` of the target, `left` and `right` is different.

Reply:
It should actually be centered.

![image](https://user-images.githubusercontent.com/7966133/145022616-cea7aea6-9db3-4b08-ba9c-c6b768785398.png)


Some samples:

![image](https://user-images.githubusercontent.com/7966133/145022701-fa65a02c-1a9b-4daa-aa01-12ad6c8ca32b.png)

![image](https://user-images.githubusercontent.com/7966133/145022877-33b25149-a37c-4b09-a5a7-08104929cc8b.png)

![image](https://user-images.githubusercontent.com/7966133/145022947-5879dda7-19a6-4aa3-bf74-b47dbe33f7cb.png)

Reply:
This is not Popper or Tippy like library this component is made for interactive contents not tooltips. Popper and Tippy is aiming to be a `tooltip` first, by the way there is `centered` prop.